.github/workflows/security.yml

```diff
@@ -0,0 +1,63 @@
+name: Security Scanning
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+  push:
+    branches:
+      - main
+      - leordev/* # todo: remove this line before merging
```

Comment:
Just a reminder to remove before merging!

Reply:
+1